assets/src/edit-story/components/canvas/framesLayer.js

```diff
@@ -88,20 +93,15 @@ function FramesLayer() {
             return <FrameElement key={id} element={{ id, ...rest }} />;
           })}
         <Selection />
-        {Boolean(draggingResource) && isDropSource(draggingResource.type) && (
-          <FrameSidebar>
-            <Hint>
-              {sprintf(
-                /* translators: %s: keyboard shortcut. */
-                __(
-                  'Drop targets are outlined in blue. Use %s to disable snapping.',
-                  'web-stories'
-                ),
-                prettifyShortcut('mod')
-              )}
-            </Hint>
-          </FrameSidebar>
-        )}
+        {Boolean(draggingResource) &&
+          isDropSource(draggingResource.type) &&
+          Object.keys(dropTargets).length && (
```

Comment:
That needs to be `length > 0`, otherwise we will end up with `0` in the output. TBH I can never remember those rules beyond a basic `false` value, so I think this is just safer.